src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] colorsArray = new String[]{"red", "blue", "black", "white", "yellow"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
```suggestion
    private static final String[] colorsArray = new String[]{"red", "blue", "black", "white", "yellow"};
    private Random random = new Random();

    public String getRandomColor() {
```
рандом краще винести в клас з методу щоб не створювати новий об'єкт при кожному виклику методу